core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -987,7 +987,7 @@ class SubmissionApiServiceSpec extends ApiServiceSpec with TableDrivenPropertyCh
     }
   }
 
-  private val userComment1000character = RandomStringUtils.random(1000)
+  private val userComment1000character = RandomStringUtils.randomGraph(1000)
```

Comment:
Before this change, the test could fail with something like:
```
o.b.d.r.webservice.RawlsApiService$ - Incorrect string value: '\xF0\xA2\x8F\x86\xF0\x90...' for column 'USER_COMMENT' at row 1
```

The description of `random` says "Characters will be chosen from the set of all characters."

See https://stackoverflow.com/questions/10957238/incorrect-string-value-when-trying-to-insert-utf-8-into-mysql-via-jdbc … the "set of all characters" can, I assume, include 4-byte characters. I assume this would fail on production too.

TBH, I'm not sure how it was passing before!

Reply:
hmm did the test fail consistently before this change? And how does that consistency compare to test runs _after_ the change?

Reply:
the test passed consistently before (else we would have blocked PRs), when it was inserting into a `latin1` collation. Now that it's inserting into `utf8` it's failing, which I believe is due to the 3-byte restriction for `utf8` collation. Something about the `latin1` collation must have allowed it, but we need `utf8` for other reasons.